rest/dbHandler.go

```diff
@@ -33,12 +33,6 @@ func (dbHandler DbHandler) Bind(r *mux.Router) {
 func (dbHandler DbHandler) insertTrackRecordPage(w http.ResponseWriter, r *http.Request) {
 	c, _, err := ProcessUploadRequest(dbHandler.Ctx.App, r)
 	if err != nil {
-		gtbackend.DebugLog(gtbackend.InternalLog{
```

Comment:
I presume this is because the error would likely be reported in other functions, though personally I would prefer to log ALL instances of where err is used, that way we can trace the problem back completely (func1Err, func2Err, insertTrackRecordPage).